File: src/Shared/FidesOS.Comunicacao/Respostas/RespostaOrdemDeServicoResumidaJson.cs
using FidesOS.Dominio.Enums;

namespace FidesOS.Comunicacao.Respostas;

public class RespostaOrdemDeServicoResumidaJson
{
  public Guid Id { get; set; }
  public string Descricao { get; set; } = string.Empty;
  public string DataAgendamento { get; set; } = string.Empty;
  public StatusOS Status { get; set; }

}

File: src/Frontend/fidesos-frontend/src/api/contracts/responses/RespostaOrdemDeServicoResumidaJson.ts
// Define o formato resumido de uma OS para a lista
export interface RespostaOrdemDeServicoResumidaJson {
  id: string; // O Guid da OS (OsIdentificacao)
  descricao: string;
  dataAgendamento: string; // Virá como string no formato ISO 8601 UTC
  status: number; // O número do Enum (0 para Pendente, 1 para Agendada, etc.)
}
